packages/node/src/utils/config.spec.ts

```diff
@@ -316,6 +316,20 @@ describe('getBaseWebpackPartial', () => {
         )
       ).toBeTruthy();
     });
+
+    it('should not add a copy-webpack-plugin if the assets option is empty', () => {
+      const result = getBaseWebpackPartial({
+        ...input,
+        assets: [],
+      });
+
+      // This test isn't great because it's hard to find CopyWebpackPlugin
+      expect(
+        result.plugins.some(
+          (plugin) => !(plugin instanceof ForkTsCheckerWebpackPlugin)
```

Comment:
You can now check for the existence of `copy-webpack-plugin` which is more preferred, see #3616